public/app/features/annotations/editor_ctrl.ts

```diff
@@ -116,25 +124,49 @@ export class AnnotationsEditorCtrl {
   }
 
   move(index: number, dir: number) {
-    // @ts-ignore
-    _.move(this.annotations, index, index + dir);
+    const { list } = this.dashboard.annotations;
+    const annotation = list[index];
+    const target: Annotation[] = [];
+
+    for (let i = 0; i < list.length; i++) {
+      const current = list[i];
+
+      if (i === index) {
+        continue;
+      }
+
+      if (dir > 0) {
+        target.push(current);
+      }
+
+      if (i === index + dir) {
+        target.push(annotation);
+      }
+
+      if (dir < 0) {
+        target.push(current);
+      }
+    }
+
```

Comment:
this seems like a lot of code for a move, is there no more terse way to do this? something like 

```
const move = (arr, from, to) => {
  const clone = [...arr];
  Array.prototype.splice.call(clone, to, 0,
    Array.prototype.splice.call(clone, from, 1)[0]
  );
  return clone;
};
```